project/Protobuf.scala

```diff
@@ -0,0 +1,173 @@
+/*
```

Comment:
side note: shouldn't this code be part of an sbt plugin?  Seems like a lot of boilerplate just to enable pb support.

Reply:
It's a copy of the one in akka/akka. For general protobuf usage one would not do it like this. User's wouldn't. This is because I wanted to use the shaded akka-protobuf-v3.